### PR TITLE
Test worker 4

### PR DESCRIPTION
**Pull request by AntonCroner**

### CODE REVIEW

**Comment by AntonCroner** [comment]
Didn't have permission to make a branch for whatever reason.

File: test-worker/model/Order/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	currency: string
+}
+
+export namespace Order {
+	export function is(value: any | Order): value is Order {
+		return (
+			value &&
+			typeof value == "object" &&
+			typeof value.id == "string" &&
+			typeof value.number == "number" &&
```
Comment (benjaminjosefsson) [inline test-worker/model/Order/index.ts:14]: typeof value.number == "number" should be "string" on line 14 or the typing of property number should be changed.

**Review by benjaminjosefsson — COMMENTED** [review]
Just a minor mistake in is() in test-worker/model/Order/index.ts otherwise it looks good.

**Review by benjaminjosefsson — APPROVED** [review]
Looks good!

**Pull request merged**